autobotAI_integrations/utils/__init__.py

```diff
@@ -50,3 +59,204 @@ def list_of_unique_elements(list_to_verify: list) -> list:
         unique_list.append(json.loads(string_json))
 
     return unique_list
+
+# Inventory Utils
+
+def change_keys(obj, convert=lambda key: key.replace('.', '_').replace('$', '-')):
```

Comment:
Is change_keys being used in all the cases? It is required because if the key has "." in it, DocumentDB will not allow saving. So Inventory or Compliance or Python Output all has to go through this.

Reply:
ChangeKeys is only applied in steampipe result